app/presenters/order_detail_presenter.rb

```diff
@@ -3,6 +3,7 @@ class OrderDetailPresenter < SimpleDelegator
   include ActionView::Helpers::NumberHelper
   include DateHelper
   include Rails.application.routes.url_helpers
+  include PriceDisplayment
```

Comment:
We don't need to do it now, but now that we have this, I'd kind of like to go through and get rid of some of the `.send(:extend, PriceDisplayment)` and replace them with the presenter.